src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs

```diff
@@ -47,7 +46,7 @@ public async Task StackTrace()
                     await Task.Delay((int)wait);
                 }
 
-                var threadStats = threadsUsage.Calculate();
+                var threadStats = threadsUsage.Calculate(threadIds.Select(int.Parse).ToHashSet());
```

Comment:
threadId parameter is not required, what will happen then?

Reply:
Empty collection.
We have similar usage here: https://github.com/ravendb/ravendb/blob/471bf17fb7057fc8a0916827a60f0c5bb1543851/src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs#L38